service/service_test.go

```diff
@@ -95,6 +95,22 @@ func TestNew(t *testing.T) {
 	})
 	require.Error(t, err)
 
+	// Incorrect string representation of whitelisted client address.
```

Comment:
Augment an existing test to check fast-fail in parsing a malformed client address in a whitelist.